### PR TITLE
Gokhan arik/codestylefiles

### PR DESCRIPTION
**Pull request by GokhanArik**

### CODE REVIEW

**Review by bgogetap — COMMENTED** [review]
Are we not planning on using checkstyle?

File: hooks/pre-push [hunk 1]
```diff
@@ -4,16 +4,24 @@
...
-RESULT=$?
-if [ $RESULT -ne 0 ]; then
- echo “failed $CMD”
+
+SIZE=1
+while read -r line
+do
+    echo "$line"
+    size=$(($SIZE + ${#line}))
+done < <(./gradlew lintDebug)
```
Comment (bgogetap) [inline hooks/pre-push]: Should this also run unit tests or are we trusting the pusher? 

Could be `./gradlew lintDebug testDebug`

File: hooks/pre-push [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+if [ -z “$commits” ]; then
+ exit 0
+fi
+
+SIZE=1
+while read -r line
+do
+    echo "$line"
+    size=$(($SIZE + ${#line}))
+done < <(./gradlew lintDebug)
```
Comment (bgogetap) [inline hooks/pre-push]: Should this also run unit tests or are we trusting the pusher? 

Could be `./gradlew lintDebug testDebug`

Comment (GokhanArik) [reply hooks/pre-push]: Sure, I will update that.

Comment (GokhanArik) [reply hooks/pre-push]: And I am researching for a way to run checkstyle automatically.

Comment (bgogetap) [reply hooks/pre-push]: Once the plugin is integrated, it should just be the gradle task that it comes with-- ./gradlew checkstyle

Or are you talking about something else?

Comment (GokhanArik) [reply hooks/pre-push]: Yes, sorry, I meant auto-formatting, not checkstyle.

**Review by bgogetap — APPROVED** [review]
LGTM

**Pull request merged**